src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -1014,4 +1238,44 @@ protected function check_upload_size( $file ) {
 		return true;
 	}
 
+	/**
+	 * Gets the request args for the edit item route.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array
+	 */
+	protected function get_edit_media_item_args() {
+		return array(
+			'rotation' => array(
+				'description' => __( 'The amount to rotate the image clockwise in degrees.' ),
+				'type'        => 'integer',
```

Comment:
@ajlende can this be constrained to a min `0` and max `360`?

Reply:
For this version, yes, that's fine. In the next, I'd prefer to not have constraints on it. It's easy enough to convert to the correct range (`( angle + 360 ) % 360 )`), but if you're rotating 270 degrees, for example, it might be more natural to say -90 instead. I'd assume the implementations for rotating an image in `WP_Image_Editor` take this into account, so converting to a specific range just seems like an extra step.